src/tests/class_tests/openms/source/MSChromatogram_test.cpp

```diff
@@ -1070,6 +1078,27 @@ START_SECTION(([MSChromatogram::MZLess] bool operator()(const MSChromatogram &a,
 }
 END_SECTION
 
+START_SECTION(void mergePeaks(MSChromatogram & other) )
+{
+  MSChromatogram a, b, c;
+  a.push_back(p1);
+  b.push_back(p2);
+  a.push_back(p3);
+  b.push_back(p5);
+  c.push_back(p1);
+  c.push_back(p2);
+  c.push_back(p4);
+  a.sortByPosition();
+  b.sortByPosition();
+  c.sortByPosition();
+  a.mergePeaks(b, true);
+  DoubleList dl = DoubleList();
```

Comment:
for brevity: `DoubleList dl = { b.getMZ() };`

Reply:
fixed